src/identity/ownership/TokenOwnershipValidator.ts

```diff
@@ -91,6 +91,12 @@ export class TokenOwnershipValidator extends OwnershipValidator {
       'to prove it belongs to you.',
       'You can remove this triple again after validation.',
     ].join(' ');
-    throw new BadRequestHttpError(errorMessage);
+    const details = {
+      quad: `<${webId}> <${SOLID.oidcIssuerRegistrationToken}> "${token}".`,
```

Comment:
Redundant?

Reply:
But why do we split in the first place?

Reply:
The main idea here was "I'm not really sure what an API user would need, so I'll just dump all the possible fields in there", but can remove some if you think it's too much

Reply:
I'd just give the quad then.